Changes

```diff
@@ -20,8 +20,9 @@ OCaml 4.07 maintenance branch
 - GPR#1915: rec_check.ml is too permissive for certain class declarations.
   (Alban Reynaud with Gabriel Scherer, review by Jeremy Yallop)
 
-- MPR#7833, GPR#1946: typecore: only 1k existential per match, not 100k
-  (Thomas Refis, report by Jerome Simeon, review by Jacques Garrigue)
```

Comment:
I find it a bit surprising that GPR#1946 has completely disappeared from the Changes file. Is it reverted by this change? I thought you reverted a part of #1609.

Reply:
As Thomas noted in MPR#7835, the limit on existentials per match was not actually doing anything before 4.07.0 because we were always using the newtype level as the binding time. Since we're checking newtype levels again with this patch, I just removed the "limit", which means that the change of the limit from 100k to 1k has essentially been reverted.